srcpkgs/gogs/files/gogs/run

```diff
@@ -1,16 +1,7 @@
 #!/bin/sh
-[ -r ./conf ] && . ./conf
-: ${USER:=gogs}
-: ${GROUP:=gogs}
-: ${HOME:=/srv/gogs}
 exec 2>&1
-cd $HOME
-export HOME
-export USER
+cd srv/gogs
+export HOME=/srv/gogs
+export USER=/gogs
```

Comment:
exporting the variables this way does not allow to overwrite them via a ./conf file.